kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -47,58 +75,89 @@ def create(
     env,
     target_path,
     jinja_file,
+    kwargs_template,
+    params,
 ):  # pylint: disable=too-many-locals,too-many-arguments
     """Create an Airflow DAG for a project"""
-    jinja_file = Path(jinja_file).resolve()
-    loader = jinja2.FileSystemLoader(jinja_file.parent)
-    jinja_env = jinja2.Environment(autoescape=True, loader=loader, lstrip_blocks=True)
-    jinja_env.filters["slugify"] = slugify
-    template = jinja_env.get_template(jinja_file.name)
-
-    package_name = metadata.package_name
-    dag_filename = f"{package_name}_dag.py"
-
-    target_path = Path(target_path)
-    target_path = target_path / dag_filename
-
-    target_path.parent.mkdir(parents=True, exist_ok=True)
-
-    pipeline = pipelines.get(pipeline_name)
-
-    dependencies = defaultdict(list)
-    for node, parent_nodes in pipeline.node_dependencies.items():
-        for parent in parent_nodes:
-            dependencies[parent].append(node)
-
-    template.stream(
-        dag_name=package_name,
-        dependencies=dependencies,
-        env=env,
-        pipeline_name=pipeline_name,
-        package_name=package_name,
-        pipeline=pipeline,
-    ).dump(str(target_path))
-
-    secho("")
-    secho("An Airflow DAG has been generated in:", fg="green")
-    secho(str(target_path))
-    secho("This file should be copied to your Airflow DAG folder.", fg="yellow")
-    secho(
-        "The Airflow configuration can be customized by editing this file.", fg="green"
-    )
-    secho("")
-    secho(
-        "This file also contains the path to the config directory, this directory will need to "
-        "be available to Airflow and any workers.",
-        fg="yellow",
-    )
-    secho("")
-    secho(
-        "Additionally all data sets must have an entry in the data catalog.",
-        fg="yellow",
-    )
-    secho(
-        "And all local paths in both the data catalog and log config must be absolute paths.",
-        fg="yellow",
-    )
-    secho("")
+    project_path = Path().cwd()
+    bootstrap_project(project_path)
+    with KedroSession.create(project_path=project_path, env=env) as session:
```

Comment:
Is this `boostrap_project` needed? The original code didn't need this function to access Kedro's pipeline, so I think it may not be necessary.